napari/layers/image/image.py

```diff
@@ -486,6 +499,23 @@ def rendering(self, rendering):
         self._rendering = Rendering(rendering)
         self.events.rendering()
 
+    @property
+    def render_as_plane(self):
+        """Should this layer be rendered as a plane.
+        If False, render as volume (default). If True, render as a plane.
+        """
+        return self._render_as_plane
+
+    @render_as_plane.setter
+    def render_as_plane(self, value: bool):
+        if self._render_as_plane is not value:
+            self._render_as_plane = value
+            self.events.render_as_plane()
```

Comment:
How about we drop `layer.rended_as_plane` and instead have `layer.plane.enabled` i.e. we add an enabled bool to the `Plane3D` evented model? This keeps the number of attributes down on the `layer`, groups all the `plane` attributes together into the `Plane3D` object and means that we don't need to add any more additional events to the `layer` - they all come from `Plane3D`.

It will also make the API expansion for the layer much easier, we can now just add a single `plane` kwarg, which takes the full dict/ plane3D object, it will make serialization cleaner too.

Thoughts @alisterburt @kevinyamauchi?

Reply:
done! All properties are now accessed via `layer.plane` which is a `PlaneManager` object (`Plane3D` with an additional `enabled` attribute)